app/models/application_choice.rb

```diff
@@ -265,6 +267,13 @@ def supplementary_statuses
     end
   end
 
+  def updated_recently_since_submitted?
+    return false if sent_to_provider_at.blank?
+
+    since = [UPDATED_RECENTLY_DAYS.days.ago, sent_to_provider_at].max
+    RecentlyUpdatedApplicationChoice.new.call(application_choice: self, since: since).exists?
```

Comment:
Can't `since` be moved inside `RecentlyUpdatedApplicationChoice`? And you can have `RecentlyUpdatedApplicationChoice.new.call(application_choice: self).exists?`

And then the method wouldn't even need to be part of the instance.